### PR TITLE
docs: change executor uri to "jinaai"

### PR DESCRIPTION
**Pull request by delgermurun**
**Goals:**

- Adapt new executor URI on docs
- ...
- ...

- ...
- ...
- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ Use prebuilt images from Hub in your Python code:
 from jina import Flow
 
 # SECRET must be provided for private Executor
```
Comment (alaeddine-13) [inline docs/fundamentals/executor/hub/use-hub-executor.md:39]: let's also handle a case for private executors. Maybe let's say now we have to login to jina hub ?

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ Use prebuilt images from Hub in your Python code:
...
-f = Flow().add(uses='jinahub+docker://<UUID>[:<SECRET>][/<TAG>]')
+f = Flow().add(uses='jinaai+docker://<USERNAME>/<NAME>[:<TAG>]')
 ```
 
-If you do not provide a `/<TAG>`, it defaults to `/latest`.
+If you do not provide a `:<TAG>`, it defaults to `/latest`.
 
 ````{important}
 To use a private Executor, you must provide the `SECRET` which is generated after `jina hub push`.
```
Comment (alaeddine-13) [inline docs/fundamentals/executor/hub/use-hub-executor.md:46]: let's also update this

**Review by JoanFM — CHANGES_REQUESTED** [review]
In the `Hub` section we should add a section about `Resources` and a clear explanation of this pattern. Also explaining that the old behavior will still be suppoorted for backwards compatibility

**Review by JoanFM — CHANGES_REQUESTED** [review]
This requires a lot of changes

**Review by JoanFM — APPROVED** [review]

**Pull request merged**